v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/BigtableChangeStreamsToGcsOptions.java

```diff
@@ -0,0 +1,187 @@
+
+    @TemplateParameter.DateTime(
+        order = 5,
+        optional = true,
+        description = "The timestamp to read change streams from",
+        helpText =
+            "The starting DateTime, inclusive, to use for reading change streams "
+                + "(https://tools.ietf.org/html/rfc3339). For example, 2022-05-05T07:59:59Z. Defaults to the "
+                + "timestamp when the pipeline starts.")
+    @Default.String("")
+    String getStartTimestamp();
+
+    void setStartTimestamp(String startTimestamp);
+
+    @TemplateParameter.DateTime(
+        order = 6,
+        optional = true,
+        description = "The timestamp to read change streams to",
+        helpText =
+            "The ending DateTime, inclusive, to use for reading change streams "
+                + "(https://tools.ietf.org/html/rfc3339). Ex-2022-05-05T07:59:59Z. Defaults to an infinite "
+                + "time in the future.")
+    @Default.String("")
+    String getEndTimestamp();
+
+    void setEndTimestamp(String startTimestamp);
+
+    @TemplateParameter.Enum(
+        order = 7,
+        enumOptions = {"TEXT", "AVRO"},
+        optional = true,
+        description = "Output file format",
+        helpText =
+            "The format of the output Cloud Storage file. Allowed formats are TEXT, AVRO. Default is AVRO.")
+    @Default.Enum("AVRO")
+    FileFormat getOutputFileFormat();
+
+    void setOutputFileFormat(FileFormat outputFileFormat);
+
+    @TemplateParameter.Duration(
+        order = 8,
+        optional = true,
+        description = "Window duration",
+        helpText =
+            "The window duration/size in which data will be written to Cloud Storage. Allowed formats are: Ns (for "
+                + "seconds, example: 5s), Nm (for minutes, example: 12m), Nh (for hours, example: 2h).",
+        example = "5m")
+    @Default.String("5m")
+    String getWindowDuration();
+
+    void setWindowDuration(String windowDuration);
+
+    @TemplateParameter.Text(
+        order = 11,
+        optional = true,
+        description = "Bigtable Metadata Table Id",
+        helpText = "Table ID used for creating the metadata table.",
+        example = "__change_stream_md_table")
+    @Default.String("")
+    String getBigtableMetadataTableId();
+
+    void setBigtableMetadataTableId(String bigtableMetadataTableId);
+
+    @TemplateParameter.Enum(
+        order = 12,
+        enumOptions = {"SIMPLE", "BIGTABLEROW"},
+        optional = true,
+        description = "Output schema format",
+        helpText = "Schema chosen for outputting data to GCS.")
+    @Default.Enum("SIMPLE")
+    BigtableSchemaFormat getSchemaOutputFormat();
+
+    void setSchemaOutputFormat(BigtableSchemaFormat outputSchemaFormat);
+
+    @TemplateParameter.Enum(
+        order = 13,
+        optional = true,
+        description = "Bigtable Charset",
+        helpText = "Bigtable Charset to be used when writing to GCS.")
```

Comment:
I think it's crucial that it's not the charset or writing, but rather a charset when you read from CBT.

This is what I have:

```
  @TemplateParameter.ProjectId(
      order = 15,
      optional = true,
      description = "BigQuery charset name when reading values and column qualifiers",
      helpText = "BigQuery charset name when reading values and column qualifiers. " +
          "Default is UTF-8")
  @Default.String("UTF-8")
  String getBigtableCharset();
```

Reply:
Good point. Changing it.

Reply:
Let's create another param getOutputCharset and default it to getBigtableCharset